### DISCUSSION TITLE
How to use IsOpen property to open dialog?

### DISCUSSION THREAD

**Discussion by nir34** [AvaloniaUtils/DialogHost.Avalonia#43] [Q&A] [answered]
```
<dialogHostAvalonia:DialogHost Identifier="MainDialogHost"IsOpen="{Binding IsOpen,Mode=TwoWay}">
    <dialogHostAvalonia:DialogHost.DataTemplates>
      <DataTemplate DataType="vm:MainWindowViewModel">
      <Border Background="Blue"></Border>
      </DataTemplate>
    </dialogHostAvalonia:DialogHost.DataTemplates>
      <Button Command="{Binding CLick}" ></Button>
  </dialogHostAvalonia:DialogHost>
```
I want to be able to open a dialog containing a border, if I use DialogHost.Show() I can open it normally, but if I use the above code, when click button,it will open an empty dialog, how should I modify my code?

**Comment by SKProCH** [comment, marked as answer]
Hello, Content must be set in `DialogHost.DialogContent`.

**Reply by nir34** [reply]
I found that the issue was caused by mistakenly writing DialogHost.Content instead of DialogHost.DialogContent.Thanks for your help!
